instantsearch/src/androidMain/kotlin/com/algolia/instantsearch/extension/Console.kt

```diff
@@ -1,24 +1,26 @@
 package com.algolia.instantsearch.extension
 
 import android.util.Log
+import com.algolia.instantsearch.InternalInstantSearch
 
-internal actual object Console {
+@InternalInstantSearch
+public actual object Console {
 
     private const val Tag = "InstantSearch"
 
-    actual fun debug(message: String, throwable: Throwable?) {
+    public actual fun debug(message: String, throwable: Throwable?) {
```

Comment:
What's the purpose of making these methods public?

Reply:
Different modules limitation, must to be public to be used, that's why the class is annotated with `@InternalInstantSearch`